specification/compute/resource-manager/Microsoft.Compute/GalleryRP/stable/2022-08-03/examples/galleryExamples/GalleryImageVersion_Create_WithVHD_UefiSettings.json

```diff
@@ -0,0 +1,404 @@
+{
+  "parameters": {
+    "subscriptionId": "{subscription-id}",
+    "resourceGroupName": "myResourceGroup",
+    "api-version": "2022-08-03",
+    "galleryName": "myGalleryName",
+    "galleryImageName": "myGalleryImageName",
+    "galleryImageVersionName": "1.0.0",
+    "galleryImageVersion": {
+      "location": "West US",
+      "properties": {
+        "publishingProfile": {
+          "targetRegions": [
+            {
+              "name": "West US",
+              "regionalReplicaCount": 1,
+              "encryption": {
+                "osDiskImage": {
+                  "diskEncryptionSetId": "/subscriptions/{subscriptionId}/resourceGroups/myResourceGroup/providers/Microsoft.Compute/diskEncryptionSet/myDiskEncryptionSet"
+                },
+                "dataDiskImages": [
+                  {
+                    "diskEncryptionSetId": "/subscriptions/{subscriptionId}/resourceGroups/myResourceGroup/providers/Microsoft.Compute/diskEncryptionSet/myOtherDiskEncryptionSet",
+                    "lun": 1
+                  }
+                ]
+              },
+              "excludeFromLatest": false
+            },
+            {
+              "name": "East US",
+              "regionalReplicaCount": 2,
+              "storageAccountType": "Standard_ZRS",
+              "excludeFromLatest": false
+            }
+          ]
+        },
+        "storageProfile": {
+          "osDiskImage": {
+            "source": {
+              "storageAccountId": "/subscriptions/{subscriptionId}/resourceGroups/myResourceGroup/providers/Microsoft.Storage/storageAccounts/{storageAccount}",
+              "uri": "https://gallerysourcencus.blob.core.windows.net/myvhds/Windows-Server-2012-R2-20171216-en.us-128GB.vhd"
+            },
+            "hostCaching": "ReadOnly"
+          },
+          "dataDiskImages": [
+            {
+              "source": {
+                "storageAccountId": "/subscriptions/{subscriptionId}/resourceGroups/myResourceGroup/providers/Microsoft.Storage/storageAccounts/{storageAccount}",
+                "uri": "https://gallerysourcencus.blob.core.windows.net/myvhds/Windows-Server-2012-R2-20171216-en.us-128GB.vhd"
+              },
+              "lun": 1,
+              "hostCaching": "None"
+            }
+          ]
+        },
+        "safetyProfile": {
+          "allowDeletionOfReplicatedLocations": false
+        },
+        "securityProfile": {
```

Comment:
As per the AdditionalPRoperties [errors](https://github.com/Azure/azure-rest-api-specs/pull/23677/checks?check_run_id=13764206324), There is no securityProfile property defined in the GalleryImageVersionProperties [swagger](https://github.com/Azure/azure-rest-api-specs/blob/475bd76df93e07b8b886e2fb1d40bd7910e6e427/specification/compute/resource-manager/Microsoft.Compute/GalleryRP/stable/2022-08-03/gallery.json#L2427), so as far as the swagger is concerned, this should not be present in the examples and the responses. 

You only added the securityProfile property to the GalleryApplicationVersionProperties [model](https://github.com/Azure/azure-rest-api-specs/pull/23677/files#diff-2d57b1e3ef500806be7f276354fc7ae66b634ce4a9ba619f0b983855859acf35R2068).